engines/python/setup/djl_python/huggingface.py

```diff
@@ -496,41 +496,96 @@ def register_adapter(inputs: Input):
     """
     Registers lora adapter with the model.
     """
+    if not is_rolling_batch_enabled(_service.hf_configs.rolling_batch):
```

Comment:
I think we should further restrict this to `lmi-dist` and `vllm`. I don't know if we have the capabilities in scheduler to do multi-lora (unless you added them)

maybe we add a method `is_lora_enabled` and validate that lmi-dist or vllm are being used. I know you also added that enable_lora param, so we can use that in validation as well.